packages/client/modules/userDashboard/components/CreditCardModal/CreditCardModal.tsx

```diff
@@ -31,20 +31,21 @@ interface Props {
   closePortal: () => void
   orgId: string
   meetingId?: string
+  invoiceListRefetch?: (refetchVariables: {}) => void
 }
 
 type Status = 'success' | 'later' | 'init'
 
 const CreditCardModal = (props: Props) => {
-  const {actionType, activeUserCount, closePortal, orgId, meetingId} = props
+  const {actionType, activeUserCount, closePortal, orgId, meetingId, invoiceListRefetch} = props
   const [status, setStatus] = useState<Status>('init')
   const atmosphere = useAtmosphere()
   const onSuccess =
     actionType === 'update'
       ? closePortal
       : () => {
-          setStatus('success')
-        }
+        setStatus('success')
```

Comment:
if `invoiceListRefetch` is replaced with `onUpgrade` you can call `onUpgrade?.()` here